<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Ajouter un employer</string>
     </attribute>
     <widget class="QLabel" name="reference">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="age">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>220</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="type">
      <property name="geometry">
       <rect>
        <x>194</x>
        <y>220</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="quantite">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="prix">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>260</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matricule">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="reference_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>matricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numcontart">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="reference_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>num_contrat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Modifier employer</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Saisir le cin de l'employer Ã  modifier :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Modifier employer</string>
      </property>
     </widget>
     <widget class="QLabel" name="type_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>matricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>200</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="reference_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>cin :</string>
      </property>
     </widget>
     <widget class="QLabel" name="quantite_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>nom :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="prix_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="matricule_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="age_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>300</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_contrat">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="type_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="type_11">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>340</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>num_contrat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Afficher un employer</string>
     </attribute>
     <widget class="QTableView" name="tabemployer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>661</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>decroissant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_19">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Supprimer un employer</string>
     </attribute>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Supprimer empoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Saisir le cin de l'empoyer Ã  supprimer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sup">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="reference_sup_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Chercher et trier employer</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>391</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Trier les empoyers</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>cin :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Chercher le empoyer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>321</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Saisir la cin des empoyers Ã  chercher:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
